egs/mobvoi/v1/local/mobvoi_data_download.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/bash
+
+# Copyright  2018  Yiming Wang
+#            2018  Johns Hopkins University (Author: Daniel Povey)
+# Apache 2.0
+
+# This script loads the speech commands dataset v0.2.
+[ -f ./path.sh ] && . ./path.sh
+
+dl_dir=data/download
+
+mkdir -p $dl_dir
+
+src_path=/export/a11/hlyu/wakeup_word_corpra/mobvoi
+
+dataset=ticmini2_dataset_20180607.zip
+if [ -d $(basename "$dataset" .zip) ]; then
```

Comment:
if [ -d **$dl_dir/**$(basename .......) ]; then

Becuase you use
```
unzip $dl_dir/$dataset -d $dl_dir
```

it is unzipped into $dl_dir